Vagrantfile

```diff
@@ -154,12 +156,15 @@ Vagrant.configure("2") do |config|
 
       # Uncomment below to enable NFS for sharing the host machine into the coreos-vagrant VM.
       #config.vm.synced_folder ".", "/home/core/share", id: "core", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+      config.vm.provision :shell, :inline => "touch /etc/fstab", :privileged => true
       $shared_folders.each_with_index do |(host_folder, guest_folder), index|
-        config.vm.synced_folder host_folder.to_s, guest_folder.to_s, id: "core-share%02d" % index, nfs: true, mount_options: ['nolock,vers=3,udp']
+        config.vm.synced_folder host_folder.to_s, guest_folder.to_s, id: "core-share%02d" % index, nfs: true, mount_options: ["#{$nfs_options}"]
+        config.vm.provision :shell, :inline => "echo \"#{$nfs_server}:#{host_folder.to_s} #{guest_folder.to_s} nfs #{$nfs_options} 0 0\" >> /etc/fstab", :privileged => true
       end
 
       if $share_home
         config.vm.synced_folder ENV['HOME'], ENV['HOME'], id: "home", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+        config.vm.provision :shell, :inline => "echo \"#{$nfs_server}:#{ENV['HOME']} #{ENV['HOME']} nfs #{$nfs_options} 0 0\" >> /etc/fstab", :privileged => true
```

Comment:
Does shell provisioning happen more than once, or does it only happen with 'vagrant provision'? If only on 'provision', then we're probably ok for now, but we should backlog a task to cleanup old entries in case we do depend on 'vagrant provision' in the future.

If it happens each time we bring it up, we may want to do that work now.